src/sentry/workflow_engine/handlers/action/notification/handler.py

```diff
@@ -25,6 +25,81 @@ class NotificationHandlerException(Exception):
     pass
 
 
+MESSAGING_ACTION_CONFIG_SCHEMA = {
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "description": "The configuration schema for a Messaging Action",
+    "type": "object",
+    "properties": {
+        "target_identifier": {"type": ["string"]},
+        "target_display": {"type": ["string"]},
+        "target_type": {
+            "type": ["integer"],
+            "enum": [*ActionTarget],
+        },
+    },
+    "required": ["target_identifier", "target_display", "target_type"],
+    "additionalProperties": False,
+}
+
+# Main difference between the discord and slack action config schemas is that
+# the target_display is null
+DISCORD_ACTION_CONFIG_SCHEMA = {
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "description": "The configuration schema for a Discord Action",
+    "type": "object",
+    "properties": {
+        "target_identifier": {"type": "string"},
+        "target_display": {
+            "type": ["null"],
+        },
+        "target_type": {
+            "type": ["integer"],
+            "enum": [*ActionTarget],
+        },
+    },
+    "required": ["target_identifier", "target_type"],
+    "additionalProperties": False,
+}
+
+TAGS_SCHEMA = {
+    "type": "string",
+    "description": "Tags to add to the message",
+}
```

Comment:
Is there only ever one tag? Are multiple values collapsed into a delimited string?